tests/models/bart/test_modeling_tf_bart.py

```diff
@@ -682,6 +701,63 @@ def test_xsum_1_1_generation(self):
         result = self.tok.batch_decode(generated_ids, skip_special_tokens=True)[0]
         assert result == EXPECTED
 
+    def test_xsum_1_1_xla_greedy_generation(self):
+        # TODO (Joao): this is temporary test, while XLA beam search is not operational. Move the XLA==non-XLA
+        # comparisons to the other tests after enabling XLA beam search.
+        # Note -- `no_repeat_ngram_size` has to be disabled, since it is not compatible with XLA
```

Comment:
Hopefully will be able to piggyback XLA tests to the other Bart tests when beam search is operational (with the caveat of `no_repeat_ngram_size`, see comment). All existing tests use beam search.

Reply:
I think it's important that we add one slow tests for XLA generate just like you did here and where `no_repeat_ngram_size` is disabled. Maybe just run it once without XLA store the result and then make sure with XLA gives the same result and then use that as the slow test?